tests/ConformanceTestOnSamples-report.txt

```diff
@@ -1,16 +1,16 @@
-# 224 pass; 349 fail; 573 total; 39.1% score
+# 67 pass; 506 fail; 573 total; 11.7% score
```

Comment:
What's your guess as to why we see new unexpected facts? Or do we need to fix https://github.com/jspecify/jspecify-reference-checker/issues/135 to find out?